librz/include/rz_cons.h

```diff
@@ -69,6 +69,19 @@ typedef struct rz_cons_bind_t {
 	RzConsGrepCallback cb_grep;
 } RzConsBind;
 
+typedef struct rz_theme_info {
+
+	char *name;
+	char *author;
+	char *license;
+	char *targetbg;
+	char *mincolordepth;
+	char *desc;
+	char *additionaldts;
+	char *refs;
+
```

Comment:
these spaces are still here.

Reply:
Fixed it manually, I am not sure why clang-format does not fix this though.